src/sidebar/panel.js

```diff
@@ -39,6 +39,7 @@ let ignoreNextLoadEvent = false;
 let ignoreTextSynced = false;
 let lastModified;
 let lastGood = null;
+let userEmail;
```

Comment:
I think we should not use a global variable here.

Reply:
Done

Reply:
@Natim , I'm so sorry in a hurry I made a forced update and missed out the review commit added by you. I'll add that back